lib/srv/usermgmt.go

```diff
@@ -218,6 +218,10 @@ func (u *HostSudoersManagement) WriteSudoers(name string, sudoers []string) erro
 		sudoersOut.WriteString(fmt.Sprintf("%s %s\n", name, entry))
 	}
 	err := u.backend.WriteSudoersFile(name, []byte(sudoersOut.String()))
+	if errors.Is(err, host.ErrInvalidSudoers) {
+		log.Warnf("encountered invalid sudoers entry in: %s", sudoersOut.String())
+		return trace.Errorf("invalid sudoers entry for login %q, inspect role's host_sudoers field or static host user's sudoers field for invalid syntax", name)
```

Comment:
Maybe a `BadParameterError`?

```suggestion
		return trace.BadParameter("invalid sudoers entry for login %q, inspect role's host_sudoers field or static host user's sudoers field for invalid syntax", name)
```

Reply:
Done!